src/libraries/System.Formats.Nrbf/tests/TypeMatchTests.cs

```diff
@@ -73,6 +73,34 @@ public void CanRecognizeGenericSystemTypes()
         Verify(new Dictionary<string, List<ValueTuple<int, short>>>());
     }
 
+    [Fact]
+    public void ThrowsForNullType()
+    {
+        List<int> input = new List<int>();
+
+        SerializationRecord record = NrbfDecoder.Decode(Serialize(input));
+
+        Assert.Throws<ArgumentNullException>(() => record.TypeNameMatches(type: null));
```

Comment:
```suggestion
        AssertExtensions.Throws<ArgumentNullException>("type", () => record.TypeNameMatches(type: null));
```

Is a better test, as it ensures that the parameter name is correct.